characteristic_notify/index.html

```diff
@@ -62,14 +62,16 @@ <h2>Step 2</h2>
     <p>Status: <span id='test_result'></span></p>
     <pre id='status'></pre>
   </div>
-  <!-- Only one of the two below will be visible. -->
-  <div id="no_bluetooth" class="no-bluetooth" style="visibility: hidden;">
-    Bluetooth is not available.
+  <!-- Only one of the three below will be visible. -->
+  <div id="bluetooth_none" class="no-bluetooth" style="visibility: hidden;">
+    Bluetooth is not present.
   </div>
-  <div id="insecure_context" class="no-bluetooth" style="visibility: hidden;">
-    Bluetooth is not available. Bluetooth requires a secure context (HTTPS).
-    For development purposes it does support HTTP, but only the on the loopback
-    adapter (i.e. "localhost").
+  <div id="bluetooth_insecure" class="no-bluetooth" style="visibility: hidden;">
+    Bluetooth requires a secure context (HTTPS). For development purposes it
+    does support HTTP, but only the on the loopback adapter (i.e. "localhost").
+  </div>
+  <div id="bluetooth_unavailable" class="no-bluetooth" style="visibility: hidden;">
+    Bluetooth is supported, but unavailable.
```

Comment:
Similarly, I would say "Web Bluetooth is supported by this browser but this device does not have Bluetooth capabilities."